web_development_101/installations/installing_ruby.md

```diff
@@ -70,10 +70,9 @@ Note: Run these commands one by one in sequence. They will not provide any outpu
 ~~~bash
 echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
 echo 'eval "$(rbenv init -)"' >> ~/.bashrc
-exit
+source ~/.bashrc
```

Comment:
This change doesn't fit with the rest of the changes. Please submit this as another pull request if you think this should be included.